inventory/group_vars/k8s-cluster.yml

```diff
@@ -71,6 +71,22 @@ kube_users:
 # Can also be set to 'cloud', which lets the cloud provider setup appropriate routing
 kube_network_plugin: calico
 
+# weave's network password for encryption
+# if null then no network encryption
+# you can use --extra-vars to pass the password in command line
+weave_password: EnterPasswordHere
+
+# Weave uses consensus mode by default
+# Enabling seed mode allow to dynamically add or remove hosts
+# https://www.weave.works/docs/net/latest/ipam/
+weave_mode_seed: false
+
+# This two variable are automatically changed by the weave's role, do not manually change these values
+# To reset values :
+# seed: unset
+# peers: unset
+weave_seed: unset
```

Comment:
No group vars can be introduced without adding a default in kubespray-defaults or network-plugin/weave defaults files. I am marking -1 for this part especially.